src/webgpu/web_platform/reftests/canvas_colorspace.html.ts

```diff
@@ -20,10 +20,40 @@ function rgba16floatFromRgba8unorm(rgba8Unorm: Uint8Array) {
   return rgba16Float;
 }
 
+type Transferable = {
+  canvas: HTMLCanvasElement | OffscreenCanvas;
+  textureData: ArrayBuffer;
+  format: GPUTextureFormat;
+  colorSpace: PredefinedColorSpace;
+  alphaMode: GPUCanvasAlphaMode;
+};
+
+function render(
+  device: GPUDevice,
+  { canvas, format, alphaMode, colorSpace, textureData }: Transferable
+) {
+  const context = canvas.getContext('webgpu') as GPUCanvasContext;
+  context.configure({
+    device,
+    format,
+    usage: GPUTextureUsage.COPY_DST | GPUTextureUsage.RENDER_ATTACHMENT,
+    alphaMode,
+    colorSpace,
+  });
+
+  const texture = context.getCurrentTexture();
+  device.queue.writeTexture(
+    { texture },
+    textureData,
+    {
+      bytesPerRow: textureData.byteLength,
+    },
```

Comment:
nit: You _should_ be able to omit bytesPerRow when height &le; 1.
```suggestion
    {},
```